plugins/inputs/p4runtime/p4runtime.go

```diff
@@ -0,0 +1,240 @@
+package p4runtime
+
+import (
+	"context"
+	"crypto/tls"
+	_ "embed"
+	"fmt"
+	"io"
+	"strconv"
+	"sync"
+
+	p4ConfigV1 "github.com/p4lang/p4runtime/go/p4/config/v1"
+	p4v1 "github.com/p4lang/p4runtime/go/p4/v1"
+	"golang.org/x/exp/slices"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	internaltls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+const (
+	defaultDeviceID = "1"
+	defaultAddress  = "127.0.0.1:9559"
+)
+
+type P4runtime struct {
+	Address      string          `toml:"endpoint"`
```

Comment:
If the docs are going to call this endpoint, let's be consistent throughout the code rather than suddenly changing from endpoint to address. As this has port info as well, it is more than an address as well.

```suggestion
	Endpoint      string          `toml:"endpoint"`
```

If you do accept this suggestion you will need to update other places as well.

Reply:
`address` -> `endpoint` throughout the code